wazo_acceptance/steps/conference.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from hamcrest import (
+    assert_that,
+    contains_inanyorder,
+)
+
+from behave import (
+    given,
+    then,
+    when,
+)
+
+
+@given('there are conference rooms with infos')
+def there_are_conference_rooms(context):
+    for row in context.table:
+        body = dict(zip(row.headings, row.cells))
```

Comment:
You can use row.as_dict()
https://github.com/behave/behave/blob/1cc411a82767fd63b4eff5bc9cff6d7867d45d17/behave/model.py#L1919